src/Raven.Server/Documents/ETL/Providers/Raven/RavenEtl.cs

```diff
@@ -100,7 +100,7 @@ protected override IEnumerator<RavenEtlItem> ConvertTimeSeriesEnumerator(Documen
         {
             return new TimeSeriesToRavenEtlItems(timeSeries, collection);
         }
-        protected override IEnumerator<RavenEtlItem> ConvertTimeSeriesDeletedRangeEnumerator(DocumentsOperationContext context, IEnumerator<TimeSeriesDeletedRangeItem> timeSeries, string collection)
+        protected override IEnumerator<RavenEtlItem> ConvertTimeSeriesDeletedRangeEnumerator(DocumentsOperationContext context, IEnumerator<TimeSeriesDeletedRangeReplicationItem> timeSeries, string collection)
```

Comment:
odd rename, epecially that this i used not only in Replication

Reply:
this is a subclass of `ReplicationBatchItem`, so the name seemed like a good fit at first.
it also matches other subclasses of `ReplicationBatchItem`, e.g. `AttachmentReplicationItem`, `CounterReplicationItem`, `TimeSeriesReplicationItem` etc.


Would you like me to revert the name change and find a different name for the newly added
`TimeSeriesDeletedRangeItem` ? e.g. `SmugglerTimeSeriesDeletedRangeItem`?

Reply:
yeah, sth like TimeSeriesDeletedRangeItemForSmuggler

Reply:
done